src/webAppCreator.ts

```diff
@@ -3,6 +3,8 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+// TODO: Will rename this file to BaseWebsiteCreator after this PR, to make changes easier to understand
```

Comment:
I feel like this is a sign the file is too big 😂: 
![screen shot 2017-10-23 at 10 40 26 am](https://user-images.githubusercontent.com/11282622/31903940-a1a4b228-b7de-11e7-879b-09d31b7e9b01.png)

We could split it up now or when we move this to the shared package

Reply:
It's mostly just a sign that GitHub doesn't handle renames well.  However, I agree that the file is too big.